jax/_src/ops/scatter.py

```diff
@@ -15,11 +15,27 @@
 # Helpers for indexed updates.
 
 
+import sys
+from typing import Any, Optional, Sequence, Tuple, Union
+
+import numpy as np
+
 from jax import lax
 from jax._src.numpy import lax_numpy as jnp
 from jax._src import util
 
 
+Array = Any
+if sys.version_info >= (3, 10):
```

Comment:
I couldn't get ellipsis type annotating, and [it appears](https://bugs.python.org/issue41810) that it won't really work until Python 3.10.  This check switch doesn't do anything yet, but in a year when JAX supports 3.10, it should save the trouble of an extra PR.